src/bloom.cpp

```diff
@@ -23,15 +23,15 @@
 
 using namespace std;
 
-void CBloomFilter::setup(unsigned int nElements, double nFPRate, unsigned int nTweakIn, unsigned char nFlagsIn, bool size_constrained) {
+void CBloomFilter::setup(unsigned int nElements, double nFPRate, unsigned int nTweakIn, unsigned char nFlagsIn, bool size_constrained, unsigned int nMaxFilterSize) {
```

Comment:
nit: why not using `uint` notation for the new arg? 
(same consideration applies to every places where `nMaxFilterSize` (or similar) has been added)

Reply:
ok i'll change that to uint32_t